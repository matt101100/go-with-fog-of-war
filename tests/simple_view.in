view
place R13
view
place G15
view
term
