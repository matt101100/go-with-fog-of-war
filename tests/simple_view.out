J10,.................................................
G15,.................................................
E15,..........................o......................
